lms/djangoapps/course_structure_api/v0/tests.py

```diff
@@ -208,9 +208,14 @@ def test_not_authorized(self):
         self.assertEqual(response.status_code, 404)
 
 
-class CourseListTests(CourseViewTestsMixin, ModuleStoreTestCase):
+class CourseListTests(CourseViewTestsMixin, SharedModuleStoreTestCase):
     view = 'course_structure_api:v0:list'
 
+    @classmethod
+    def setUpClass(cls):
```

Comment:
I did it this way because `CourseViewTestsMixin` just sub-classes `object`, so the `super()` call here won't find the `setUpClass()`. That being said, maybe I should just have it subclass `SharedModuleStoreTestCase` and delete the two test stubs it has?